features/cart/shopping_cart/resolving_cart_order_items.feature

```diff
@@ -0,0 +1,16 @@
+@shopping_cart
```

Comment:
The file name could be adjusted to the feature name

Reply:
To be honest, I thought about changing file name, not feature name, because in this scenario you just view items in the cart. `Resolving` doesn't look like an appropriate word here

Reply:
File and feature name has been changed